app/_hub/kong-inc/bot-detection/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: Bot Detection
 publisher: Kong Inc.
-version: 1.0.0
+version: 3.0.x
```

Comment:
I _think_ version can be removed when using single sourcing as it gets injected by the generator